node/cli/src/cli.rs

```diff
@@ -175,6 +175,27 @@ pub struct RunCmd {
 	#[clap(long, default_value = "300000000")]
 	pub eth_statuses_cache: usize,
 
+	/// Sets the frontier backend type (KeyValue or Sql)
+	#[arg(long, value_enum, ignore_case = true, default_value_t = FrontierBackendType::default())]
+	pub frontier_backend_type: FrontierBackendType,
+
+	// Sets the SQL backend's pool size.
+	#[arg(long, default_value = "100")]
+	pub frontier_sql_backend_pool_size: u32,
+
+	/// Sets the SQL backend's query timeout in number of VM ops.
+	#[arg(long, default_value = "10000000")]
+	pub frontier_sql_backend_num_ops_timeout: u32,
+
+	/// Sets the SQL backend's auxiliary thread limit.
+	#[arg(long, default_value = "4")]
+	pub frontier_sql_backend_thread_count: u32,
+
+	/// Sets the SQL backend's query timeout in number of VM ops.
+	/// Default value is 200MB.
+	#[arg(long, default_value = "209715200")]
```

Comment:
I'm confused about what these units have to do with each other:
* VM ops
* Timeout
* bytes

Being creative, I can come up with something like "timeout expressed in VM ops once they accumulate to some number of bytes," but I'm sure I have the wrong idea...

Can you explain this to me, and perhaps consider clarifying the comments here?

Reply:
Maybe @nbaztec can give more context on this?

Reply:
This is related to the sqlite driver, and the new sql backend that was added to frontier. The driver records a timeout in VM operations in lieu of time.